lib/game.rb

```diff
@@ -0,0 +1,76 @@
+require_relative 'player'
+
+class Game
+  attr_reader :player_one
+  attr_reader :player_two
+  attr_reader :result
+  attr_reader :current_turn
+  attr_reader :spock_game
+
+  def initialize(player_one, player_two, spock_game)
+    @player_one = player_one
+    @player_two = player_two
+    @current_turn = @player_one
+    @spock_game = spock_game
+  end
+
+  def spock_game?
+    @spock_game
+  end
+
+  def spock_update(value)
+    @spock_game = value
+  end
+
+  def switch_turn
+    if @current_turn == @player_one
+      @current_turn = @player_two
+    elsif @current_turn == @player_two
+      @current_turn = @player_one
+    end
+  end
+  
+  def opponent
+    if @current_turn == @player_one
+      @player_two
+    elsif @current_turn == @player_two
+      @player_one
+    end
+  end
+
+  def outcome(player_one, player_two)
```

Comment:
Probably a when-case statement to get rid of the if-else-nesting or a hash-lookup table that returns the winner for each case would be leaner